scripts/testnet/start.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+export ELRONDTESTNETSCRIPTSDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+source "$ELRONDTESTNETSCRIPTSDIR/variables.sh"
+source "$ELRONDTESTNETSCRIPTSDIR/include/config.sh"
+source "$ELRONDTESTNETSCRIPTSDIR/include/build.sh"
+source "$ELRONDTESTNETSCRIPTSDIR/include/nodes.sh"
+source "$ELRONDTESTNETSCRIPTSDIR/include/tools.sh"
+
+prepareFolders
+
+# Phase 1: build Seednode and Node executables
+buildConfigGenerator
+buildSeednode
+buildNode
+buildProxy
+buildTxGen
+
+# Phase 2: generate configuration
+generateConfig
+copyConfig
+
+copySeednodeConfig
+updateSeednodeConfig
+
+copyNodeConfig
+updateNodeConfig
+
+copyProxyConfig
+updateProxyConfig
+copyTxGenConfig
+updateTxGenConfig
+
+# Phase 3: start the Seednode
+startSeednode
+showTerminalSession "elrond-tools"
+echo "Waiting for the Seednode to start ($SEEDNODE_DELAY s)..."
+sleep $SEEDNODE_DELAY
+
+# Phase 4: start the Observer Nodes and Validator Nodes
+startObservers
+startValidators
+showTerminalSession "elrond-nodes"
+echo "Waiting for the Nodes to start ($NODE_DELAY s)..."
+sleep $NODE_DELAY
+
+# Phase 5: start the Proxy
+startProxy
+echo "Waiting for the Proxy to start ($PROXY_DELAY s)..."
+sleep $PROXY_DELAY
```

Comment:
maybe move the waiting in startProxy, same for the seednode

Reply:
Waiting for a command to finish should not be part of the command, and it would make you dig for the `sleep` command into `include/tools.sh`. This way, `start.sh` remains a list of atomic actions (at least this was the intention).